src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -574,7 +574,12 @@ public function getXEditableType($type)
             ), E_USER_DEPRECATED);
         }
 
-        return $this->xEditableTypeMapping[$type] ?? false;
+        /* @phpstan-ignore-next-line */
+        if (null === $this->xEditableExtension) {
```

Comment:
Instead adding `@phpstan-ignore-next-line`

You have to declare 
```
/**
 * NEXT_MAJOR: Remove this property
 *
 * @var XEditableExtension|null
 **/
private $xEditableExtension;
```
Same for others properties.